### PR TITLE
add guessing game

### LINKED ISSUE #39: COS - Add a recap when you "finish" one side of a screen
Currently if you go through one or the other of a given side of an onboarding screen (un- or consentful (c8l?)) it just advances to the next screen. Instead when you complete a side of the screen it should give you a popup summing up what you just went through in the screen.

~~A user should also have to complete both sides of a given screen to advance.~~ done in #53

### PR DESCRIPTION
**Pull request by ryanatkn**
closes #39 

Adds a little quiz at the end of each screen. I originally made the prompt randomize between `'less' || 'more'` but it didn't seem useful, because "less" means worse and "more" means better, so it doesn't make the user think any more than leaving it out. The first implementation I tried randomized between `'less' || 'more'` along with `'consentful' || 'unconsentful'`, but it felt like confusing word games, and it's better to use the actual principle in question. I think there's a similar problem if we negate each term, e.g. `'specific' || 'nonspecific'`, but it's probably a little better overall? I kept it simple because I wasn't sure.

It's deployed via `gro deploy --branch guessing-game`.

Also restructures the modules to be under `src/lib/sketch/onboard/` instead of being top-level under `src/lib/`, which seems better except for things that get reused elsewhere. Sets the pattern of co-locating all sketches under `src/lib/sketch/`.

### CODE REVIEW

File: src/lib/sketch/onboard/Sides_Complete.svelte [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+	<div class="icon">
+		{#if correct === null}🤔{:else if correct}😊{:else}😐{/if}
+	</div>
+	<div class="prompt">
+		{#if correct === null}
+			which feels {principle.type}?
+		{:else if correct}
+			we think so too ✓
+		{:else}
+			oww really? ✗
```
Comment (greatbacon) [inline src/lib/sketch/onboard/Sides_Complete.svelte:26]: When this part of pops up in the flow it feels kind of condescending. Maybe a softer "We think the other flow better captures {principle.type}" or something to that effect?

**Review by greatbacon — APPROVED** [review]
I had a couple of notes, but overall this looks amazing! 🚢

File: src/lib/sketch/onboard/Begin.svelte [hunk 2]
```diff
@@ -13,19 +13,14 @@
 	</h1>
 	<p>This sketch is a proof of concept, side-by-side comparsion between two onboarding flows:</p>
 	<ul>
```
[the remark this answers is not in the document]
Comment (ryanatkn) [reply src/lib/sketch/onboard/Begin.svelte:15]: Agree I was intending for us to make a second pass at the first and last pages, I'll revert some of my changes.

**Comment by ryanatkn** [comment]
thanks :smile: it's got some tweaks from your feedback, can change more before merging

**Pull request merged**